### PR TITLE
https://issues.redhat.com/browse/ACM-15598

### PR DESCRIPTION
**Pull request by dockerymick**

### CODE REVIEW

File: release_notes/acm_whats_new.adoc [hunk 1]
```diff
@@ -82,7 +82,7 @@ For other Application topics, see link:../applications/app_management_overview.a
 
 * Grafana is updated to version 11.1.5. See link:../observability/design_grafana.adoc#using-grafana-dashboards[Using Grafana dashboards].
 
-* You can now use the _Advanced search_ from the console by selecting the *Advanced search* drop-down button. Specify your query and receive results that match the exact strings that you enter and range-based search parameters. See link:../observability/search_console.adoc#search-customization[Search customization and configurations].
+* You can now use the _Advanced search_ from the console by selecting the *Advanced search* drop-down button. Specify your query and receive results that match the exact strings that you enter and range-based search parameters. See link:../console/search_console.adoc#search-customization[Search customization and configurations].
```
Comment (swopebe) [inline release_notes/acm_whats_new.adoc]: Either "the _Advanced search_ option" or "use _Advanced search_" (I prefer the first, but you go in to detail later...so up to you.

File: troubleshooting/trouble_shared_memory.adoc [hunk 2]
```diff
@@ -61,4 +61,4 @@ oc delete pod search-postgres-xyz search-api-xzy
 oc get cm search-postgres -n open-cluster-management -o yaml
 ----
 
-See link:../observability/search_console.adoc#search-customization[Search customization and configurations] for more information on adding environment variables.
+See link:../console/search_console.adoc#search-customization[Search customization and configurations] for more information on adding environment variables.
```
Comment (swopebe) [inline troubleshooting/trouble_shared_memory.adoc:77]: this appears grayed out, so check format?

**Review by swopebe — APPROVED** [review]
couple comments

File: release_notes/acm_whats_new.adoc (hunk 1 above)
Comment (dockerymick) [inline release_notes/acm_whats_new.adoc]:
```suggestion
* You can now use the _Advanced search_ option from the console by selecting the *Advanced search* drop-down button. Specify your query and receive results that match the exact strings that you enter and range-based search parameters. See link:../console/search_console.adoc#search-customization[Search customization and configurations].
```

**Comment by dockerymick** [comment]
@swopebe thanks for the review and approval Brandi!

**Pull request merged**